index.html

```diff
@@ -316,5 +316,65 @@ <h3 class="heading-link">
         </ul>
       </div>
     </section>
+
+    <section class="contact-cont-form">
+      <form
```

Comment:
Please use all the decoration images as the background of the form as can be seen in the design, and make sure that they are positioned as per the design.